File: SchoolApp.Models/DataModels/ExamSubject.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SchoolApp.Models.DataModels
{
    [Table("ExamSubject")]
    public class ExamSubject
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ExamSubjectId { get; set; }
        public int ExamScheduleStandardId { get; set; }
        public int SubjectId { get; set; }
        public int ExamTypeId { get; set; }
        public DateTime? ExamDate { get; set; } = DateTime.Now;
        public TimeSpan? ExamStartTime { get; set; }
        public TimeSpan? ExamEndTime { get; set; }
        public virtual Subject? Subject { get; set; }
        public virtual ExamType? ExamType { get; set; }
        public virtual ExamScheduleStandard? ExamScheduleStandard { get; set; }
    }
}

File: SchoolAppClient.NG/src/app/Models/exam-subject.ts
import { ExamScheduleStandard } from "./exam-schedule-standard";
import { Examtype } from "./examtype";
import { Subject } from "./subject";

export class ExamSubject {
  examSubjectId: number = 0;
  examScheduleStandardId !: number;
  subjectId !: number;
  examTypeId !: number;
  examDate: Date = new Date();
  examStartTime!: Date;
  examEndTime!: Date;
  subject!: Subject;
  examType!: Examtype;
  examScheduleStandard!: ExamScheduleStandard;
}
